MaterialDesignThemes.Wpf/DrawerClosingEventArgs.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+
+namespace MaterialDesignThemes.Wpf
+{
+    public class DrawerClosingEventArgs : RoutedEventArgs
```

Comment:
Rather than setting RoutedEvent directly in this constructor you can pass it into the base class.
```
 public DrawerClosingEventArgs(Dock dock, RoutedEvent routedEvent) : base(routedEvent)
```